services/data/postgres_async_db.py

```diff
@@ -510,6 +537,17 @@ async def update_heartbeat(self, flow_id: str, run_id: str):
         return DBResponse(response_code=result.response_code,
                           body=json.dumps(body))
 
+    async def update_run_tags(self, flow_id: str, run_id: str, run_tags: list, cur: aiopg.Cursor = None):
+        run_key, run_value = translate_run_key(run_id)
+        filter_dict = {"flow_id": flow_id,
+                       run_key: str(run_value)}
+        set_dict = {"tags": "'" + json.dumps(run_tags) + "'"}  # TODO we can/should do better than manual quoting
```

Comment:
May be a good idea to sort them before storing. Not sure if it makes any practical difference but its nice from debuggability perspective that e.g. after removing and adding a tag you'd have exact same value in the db.

Reply:
Discussed offline.  My view is to follow [this](https://www.hyrumslaw.com/) here.  I get the point about potentially improved debuggability.

My preference is to leave it unordered until we are ready to commit to promising this in the public contract of the API.